completion/waffle.py

```diff
@@ -87,3 +83,17 @@ def visual_progress_enabled(course_key):
         return waffle_flag().is_enabled(course_key)
 
     return True
+
+
+def site_disables_visual_progress():
+    """
+    Returns True if and only if the SiteConfiguration for the current Site
+    explicitly disables visual progress.
+    """
+    try:
+        current_site = get_current_site()
+        if current_site.configuration.get_value(ENABLE_SITE_VISUAL_PROGRESS, True) is False:
+            return True
```

Comment:
Why the explicit return `True` here? Are you thinking `return current_site.configuration.get_value(ENABLE_SITE_VISUAL_PROGRESS, True) is False" not clear enough?

Reply:
Fair point, I just tightened this up a little.